config/testdata/gh1204_config

```diff
@@ -0,0 +1,14 @@
+[default]
+region=us-west-2
+external_id=FooBar
+role_session_name=FooSessionName
+role_arn=foofoofoofoofoofoofoofoofoofoofoofoofoofoo
+duration_seconds = 43200
+source_profile=A
```

Comment:
Just as an extra check, could you add two additional profiles in this file - one with mixed case & other lower case profile name that may potentially have conflict with the profile name used for source_profile. 

Could also add an error case to assert a descriptive error is thrown if an exact match isn't available for the profile specified as source_profile.